src/gui/editors/PianoRoll.cpp

```diff
@@ -1870,7 +1870,7 @@ void PianoRoll::mouseReleaseEvent( QMouseEvent * me )
 		{
 			// select the notes within the selection rectangle and
 			// then destroy the selection rectangle
-			computeSelectedNotes( 
```

Comment:
@tresf these lines actually had a trailing space, so atom (my text editor) fixed that.